_stbt/core.py

```diff
@@ -1143,7 +1143,8 @@ def save_frame(image, filename):
     cv2.imwrite(filename, image)
 
 
-def wait_until(callable_, timeout_secs=10, interval_secs=0, stable_secs=0):
+def wait_until(callable_, timeout_secs=10, interval_secs=0, predicate=None,
+               stable_secs=0):
```

Comment:
`stable_secs` hasn't been released yet, so there's no backward compatibility issue there. I'll change it anyway so that it comes last, though it seems unlikely that anyone would pass all of these as positional parameters.